### DISCUSSION TITLE
Database Dispose error

### DISCUSSION THREAD

**Discussion by akbarbd** [meysamhadeli/booking-microservices#44] [Show and tell]
### When data save simultaneously, then some time showing this error.

> System.ObjectDisposedException: 'Cannot access a disposed context instance. 
>       A common cause of this error is disposing a context instance that was resolved
>       from dependency injection and then later trying to use the same context instance 
>       elsewhere in your application. This may occur if you are calling 'Dispose' on the context instance, 
>       or wrapping it in a using statement. If you are using dependency injection, you should let the dependency
>       injection container take care of disposing context instances.
>       Object name: 'MyDbDbContext'.'

I'm trying to find out this error, I can't.

**Comment by meysamhadeli** [comment]
> ### When data save simultaneously, then some time showing this error.
> > System.ObjectDisposedException: 'Cannot access a disposed context instance.
> > A common cause of this error is disposing a context instance that was resolved
> > from dependency injection and then later trying to use the same context instance
> > elsewhere in your application. This may occur if you are calling 'Dispose' on the context instance,
> > or wrapping it in a using statement. If you are using dependency injection, you should let the dependency
> > injection container take care of disposing context instances.
> > Object name: 'MyDbDbContext'.'
> 
> I'm trying to find out this error, I can't.

Could you give me more information? Which endpoint did you call? Please send me more info that I can debug this issue.

Thanks...

**Reply by akbarbd** [reply]
Brother,
Here is my project .
Also i face a problem when simultaneously data insert into buyer endpoint then one of them consumer  is missing for inserted data. 

[Safe.zip](https://github.com/meysamhadeli/booking-microservices/files/9497599/Safe.zip)

Thanks

**Reply by meysamhadeli** [reply]
Please create this problem in a fork of this project and sent it to me, it's easier for me for checking this problem.

Thanks...

**Reply by akbarbd** [reply]
Ok,
